### PR TITLE
Add the ChemMaster and CondiMaster Neo

### PR DESCRIPTION
**Pull request by pyrosshade**
Add the ChemMaster and CondiMaster Neo
Fulfills item 4 of #6281 bounty issue

### Changelog:
CL: [Improvement] Updates ChemMaster with functionality
CL: [New] Any ReagentContainer can be hand placed in the ChemMaster as a container. This will need tweaking for balance and sensibility
CL: [New] ChemMaster starts with two 50u Beakers for it's buffer
CL: [New] ChemMaster can create pills, bottles and patches filled from reagents in it's buffer
CL: [New] ChemMaster can transfer 1, 5, 10, all, or custom integer reagent amounts, as well as analyze a given reagent, giving a short description of the reagent
CL: [New] ChemMaster can create products with custom names (character limit 50)
CL: [New] Adds Condimaster Neo, effectively a service department reskin of the ChemMaster
CL: [New] Condimaster Neo creates condiment bottles and condiment packets instead of what ChemMaster creates
CL: [Improvement] Changes sprite filtering of SquareButtonvs16x16.png to POINT filtering for more crisp and/or faithful rendering of the given buttons

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Chemistry/ChemMaster.cs [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+using UnityEngine;
+using Systems.Electricity;
+using Items;
+
+namespace Chemistry
+{
+	/// <summary>
+	/// Main component for ChemMaster, or Chemical Master™.
+	/// </summary>
+	public class ChemMaster : NetworkBehaviour, ICheckedInteractable<HandApply>, IAPCPowerable
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Objects/Chemistry/ChemMaster.cs]: Hmm couldn't find anything that would require inheriting from `NetworkBehaviour`. Did I miss something?
```suggestion
	public class ChemMaster : MonoBehaviour, ICheckedInteractable<HandApply>, IAPCPowerable
```

Comment (pyrosshade) [reply UnityProject/Assets/Scripts/Objects/Chemistry/ChemMaster.cs]: That is a fair point, between Interaction Management and NetUI already handling the messaging.

**Comment by ThatDan123** [comment]
Codacy got two complaints

**Comment by corp-0** [comment]
both valid issues btw

**Comment by ThatDan123** [comment]
Have you built tested with headless and client?

**Comment by NoooneyDude** [comment]
I'm testing it now.

**Comment by NoooneyDude** [comment]
Works, a few little issues that can be fixed later. Thank you!

I've posted the issues in Discord #chemistry channel.

**Pull request merged**